install_config/topics/ocp_osp_node_registration.adoc

```diff
@@ -0,0 +1,53 @@
+
+[subs=+quotes]
+----
+$ cat ~/repos.yml
+---
+- name: Enable the proper repositories for OpenShift installation
+  hosts: OSEv3
+  become: yes
+  tasks:
+  - name: Register with activationkey and consume subscriptions matching Red Hat Cloud Suite or Red Hat OpenShift Container Platform
+    redhat_subscription:
+      state: present
+      activationkey: *<key-name>*
+      org_id: *<orig_id>*
+      pool: '^(Red Hat Cloud Suite|Red Hat OpenShift Container Platform)$'
+
+  - name: Disable all current repositories
+    rhsm_repository:
+      name: '*'
+      state: disabled
+
+  - name: Enable Repositories
+    rhsm_repository:
+      name: "{{ item }}"
+      state: enabled
+    with_items:
+      - rhel-7-server-rpms
+      - rhel-7-server-extras-rpms
+      - rhel-7-server-ansible-2.5-rpms
+      - rhel-7-server-ose-3.10-rpms
+      - rhel-7-fast-datapath-rpms
```

Comment:
It has been removed since OCP-3.10 (SDN run as static pod).

Reply:
What has been removed?

Reply:
rhel-7-fast-datapath-rpms